bylaws/003-membership.md

```diff
@@ -128,5 +128,119 @@ available.
 The expulsion of a Member Project requires a vote in accordance with the
 [Voting Protocol bylaw][voting].
 
+FIG Secretary
+-------------
+
+### Overarching role
+
+The primary responsibility of the FIG Secretary is to serve as an impartial
+administrator of the FIG. They serve at the member projects pleasure but act as
+independent and impartial adjudicators. FIG Secretaries will also represent the
+FIG as a whole to the general community and public on matters relating to the
+FIG's activities.
+
+Whilst there are a number of defined functions below but they should also perform
+other duties as required. As the role has continuity (in that there will always
+be a post holder) and redundancy (There are a number of post holders in case of
+the absence of one) it can be ensured that responsibilities assigned to the
+secretary will always be completed and therefore administrative responsibilities
+such as vote management should always be assigned to the FIG secretary.
+
+All FIG secretaries are expected to remain impartial and professional when acting
+in an official capacity as secretary and should also remain aware that even when
+not acting in an official capacity, their actions reflect back on the PHP FIG and
+the FIG must not be brought into disrepute.
+
+### Defined functions
+
+There are a number of defined functions that the FIG secretaries are expected to
+complete but they may also, within their remit defined in the above overaching
+role above, perform other duties as nessasary.
+
+* Keeping the website, twitter and other marketing mediums up-to-date
+* GitHub organization administration
+* Tallying votes
+* Tracking member project activity, and marking projects as inactive
+* Ensuring bylaws are being followed
+* Clarifying any interpretation of bylaw text (If there is lack of consensus
+between FIG secretaries then it must be put to a full vote according to the
+standard [Voting Protocol bylaw][voting]).
+* Ensure that relevant marketing mediums (e.g. Twitter and Facebook) are kept
+professional, up to date and impartial.
+* Moderate discussions on github, the mailing list and IRC channels to ensure
+that an appropriate environment is maintained
```

Comment:
Should add a task here:

* Should there be a FIG meeting at a conference or other ad hoc gathering any Secretary in attendance should take notes to report back to the mailing list.